tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml

```diff
@@ -139,7 +139,6 @@
             <class name="org.ballerinalang.test.expressions.invocations.StringFunctionInvocationExprTest"></class>
             <class name="org.ballerinalang.test.expressions.invocations.PackageInitInvocationTest"></class>
             <class name="org.ballerinalang.test.expressions.unaryoperations.UnaryExprTest"></class>
-            <class name="org.ballerinalang.test.imports.ImportsTest"></class>
```

Comment:
Why is this test removed?

Reply:
Because it is already enabled under the package.